platform/representation/src/gestures/DropGesture.js

```diff
@@ -90,25 +107,61 @@ define(
                 }
             }
 
+            function createVirtualPanel(base, overlayId){
```

Comment:
In the case where a telemetry object has been dragged onto another telemetry object, creates a new temporary panel object, and adds them both to its composition.

All of this 'virtual panel' stuff needs to be externalized somewhere, but I'm not sure about the best approach. Perhaps as a new action that is triggered when a telemetry object is dropped on another telemetry object?